vendor/data/country_to_region_mapping.rb

```diff
@@ -0,0 +1,253 @@
+﻿REGION_CODES = {
+"AF" => "679",
+"AX" => "89",
+"AL" => "89",
+"DZ" => "189",
+"AS" => "889",
+"AD" => "89",
+"AO" => "289",
+"AI" => "380",
+"AQ" => "",

```

Comment:
Do we know that these missing countries are no in the country list?

Reply:
Good point... Maybe I should add a default to "Developing countries, unspecified" for the ones which are nil, given this is the default in the application 🤔

Reply:
I've checked all the "" countries and none of them appear on the DAC country list, so I think we're OK here!